lib/simulator-xcode-9.js

```diff
@@ -51,8 +52,14 @@ class SimulatorXcode9 extends SimulatorXcode8 {
   }
 
   async shutdown () {
+    await restoreTouchEnrollShortcuts();
```

Comment:
we need to make sure that no actions we do in the OS can affect other tests running in parallel